src/common/components/entry-list/index.tsx

```diff
@@ -146,15 +149,27 @@ export class EntryListContent extends Component<Props, State> {
 
               let isPostMuted =
                 (activeUser && activeUser.username && mutedList.includes(e.author)) || false;
-              l.push(
-                <EntryListItem
-                  key={`${e.author}-${e.permlink}`}
-                  {...this.props}
-                  entry={e}
-                  order={i}
-                  muted={isPostMuted}
-                />
-              );
+              if (location.pathname.includes("/promoted")) {
```

Comment:
this page should be accessible even with no user is logged in.

Reply:
@feruzm  done. kindly check this.

https://user-images.githubusercontent.com/106739598/204727655-ecbc6bb0-99aa-4454-adf5-9fed0660c268.mp4